test-utils/src/indexer/test_indexer.rs

```diff
@@ -288,9 +288,9 @@ impl<const INDEXED_ARRAY_SIZE: usize, R: RpcConnection + Send + Sync + 'static>
             .find(|x| x.accounts.merkle_tree == pubkey)
             .unwrap();
 
-        let new_low_element = context.new_low_element.unwrap();
-        let new_element = context.new_element.unwrap();
-        let new_element_next_value = context.new_element_next_value.unwrap();
+        let new_low_element = context.new_low_element.clone().unwrap();
```

Comment:
Unfortunately not.

```
error[E0507]: cannot move out of `context.new_low_element` which is behind a shared reference

   --> test-utils/src/indexer/test_indexer.rs:291:31
    |
291 |         let new_low_element = context.new_low_element.unwrap();
    |                               ^^^^^^^^^^^^^^^^^^^^^^^ -------- `context.new_low_element` moved due to this method call
    |                               |
    |                               help: consider calling `.as_ref()` or `.as_mut()` to borrow the type's contents
    |                               move occurs because `context.new_low_element` has type `std::option::Option<IndexedElement<usize>>`, which does not implement the `Copy` trait

```